docker/pom.xml

```diff
@@ -52,8 +52,6 @@
         <jakarta.transaction-api.version>2.0.1-RC1</jakarta.transaction-api.version>
         <!-- Jakarta Connectors -->
         <jakarta.resource-api.version>2.1.0</jakarta.resource-api.version>
-        <!-- Jakarta Batch -->
```

Comment:
We should leave docker/pom.xml as is also as that is used for generating signature map files.